CONTRIBUTING.md

```diff
@@ -1 +1 @@
-Please see [our guide to contributing](docs/pages/contributing.md).
+Please see [our guide to contributing](docs/guides/contributing.md).
```

Comment:
I'm going to revert this change since it's already been fixed in `main` and is causing a conflict at the moment

```suggestion
Please see [our guide to contributing](docs/pages/contributing.md).
```